src/components/LicenseCode.vue

```diff
@@ -1,28 +1,46 @@
 <template>
```

Comment:
I edited LicenseIcons css only looking at this component, so I added margin-left of 3px to the icons so that they are not too close to the license name in RichText code. Now I see LicenseIcons are used in the HelpSection as well, so because the left margin is local to LicenseCode component,  it would be better to add styling here, so that the styling in other components is not affected.
```
.license-text .photo-license-icons {
    height: 1.4rem!important;
    **margin-left: 3px;**
}
```